pkg/utils/net-attach-def_test.go

```diff
@@ -129,10 +129,11 @@ var _ = Describe("Netwok Attachment Definition manipulations", func() {
 		}
 		devInfo := v1.DeviceInfo{
 			Type:    "pci",
-			Version: "v1.0.0",
+			Version: "1.1.0",
```

Comment:
shouldn't this have the `v` ? The PR to the main branch had: https://github.com/k8snetworkplumbingwg/network-attachment-definition-client/pull/49/files#diff-d03ed112d9ea34464638a1648a221afde72420c185870e113962b035c6e8c4aaR148

Reply:
no, spec define version without "v" https://github.com/k8snetworkplumbingwg/device-info-spec/blob/main/SPEC.md?plain=1#L293.  All components are using types.DeviceInfoVersion constant which has no "v". So, wrong version format was used in the test. When I was fixing test for the main branch I didn't notice this.

Reply:
I decided to keep "v" here, because for me it looks a bit strange if cherry-picked commit will contain other value then original commit.

Reply:
Yes, I agree.